src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -4,18 +4,29 @@ sidebar_order: 106
 description: "Learn more about taking screenshots when an error occurs. Sentry pairs the screenshot with the original event, giving you additional insight into issues."
 supported:
   - android
+  - apple.ios
+notSupported:
+  - apple.macos
+  - apple.tvos
+  - apple.watchos
 ---
 
 <Note>
 
-The Screenshots feature is available in Beta. Beta features are still in-progress and may have bugs. We recognize the irony.
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
 
 </Note>
 
 When a user experiences an error, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
 
 This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently, this feature will be a best effort solution.
 
+<PlatformSection supported={["apple.ios"]}>
+
+On iOS, it's not possible to take a screenshot during hard crashes, only manually captured errors or exceptions will contain a screenshot.
```

Comment:
I wouldn't mention that it's not possible. It might be somehow, but we don't support it. 
```suggestion
On iOS, only manually captured errors or exceptions will contain a screenshot.
```